app/validators/concerns/waste_exemptions_engine/validates_presence.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  module ValidatesPresence
```

Comment:
It's a name thing! If you look in `app/models/concerns/waste_exemptions_engine` you'll see we have a convention of naming our concerns `CanDoThing`. So in this case something along the lines of `CanValidatePresence` would bring it into line with that convention.

Reply:
I'll fix this now.

Reply:
Fixed in bcdf764